MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceModel.cpp

```diff
@@ -805,8 +752,10 @@ TomographyIfaceModel::loadFITSImage(const std::string &path) {
   }
 }
 
-void TomographyIfaceModel::logMsg(const std::string &msg) {
-  g_log.notice() << msg << '\n';
+void TomographyIfaceModel::logMsg(const std::string &msg) { g_log.notice(msg); }
```

Comment:
The presenter doesn't have access to the `g_log`, all the logging is being done through the model